Library/Homebrew/cmd/help.rb

```diff
@@ -21,6 +20,7 @@
 
 Further help:
   man brew
+  brew help COMMAND
```

Comment:
They may have got this just from running `brew` so I think it's almost just a question of documentation of command arguments.